lib/shopify_api/meta.rb

```diff
@@ -5,6 +5,7 @@ class Meta < ActiveResource::Base
     self.site = "https://app.shopify.com/services/"
     self.element_name = 'api'
     self.primary_key = :handle
+    self.timeout = 90
```

Comment:
This is too high; if the api doesn't respond in a second or so something bigger has gone wrong, and waiting another 89s is likely just wasting time. I'd probably go with 5s to be very conservative.

Reply:
sure. I just grabbed this from ShopifyAPI::Base, but I suppose a longer timeout makes sense there given the api. This one endpoint should be pretty zippy always